content/applications/general.rst

```diff
@@ -16,130 +16,70 @@ General
-    :ref:`duplicate <duplicate_online>` of your database before making any
-    changes (*especially* installing/uninstalling apps).
-
-In your Odoo instance click on **Settings**; in this app, you will be
-able to see how many applications you have installed. Click on **Browse
-Apps** to access the list of your installed applications.
-
-+----------------+----------------+
-|   |settings|   | |browse_apps|  |
-+----------------+----------------+
-
-In your applications' dashboard, you will see all the icons of your
-applications. Click on the application you want to uninstall. Then, on
-the form of the application, click on **Uninstall**.
+**Odoo** is a suite of open source business apps that cover all your company needs: :doc:`CRM
+<sales/crm>`, :doc:`eCommerce <websites/ecommerce>`, :doc:`accounting <finance/accounting>`,
+:doc:`inventory <inventory_and_mrp/inventory>`, :doc:`point of sale <sales/point_of_sale>`,
+:doc:`project management <services/project>`, etc. Its unique value proposition is to be at the same
+time *very easy* to use and *fully integrated*.
+
+This section of the documentation covers all functional aspects that aren't app-specific. You can
+find, for example, information about:
+
+- :doc:`Users settings <general/users>` and their :doc:`access rights <general/users/access_rights>`
+- :doc:`Export and import data <general/export_import_data>`
+- :doc:`Payment Acquirers <general/payment_acquirers>` allow your customers to pay you with their
+  favorite payment method.
+- :doc:`Google Sign-In <general/auth/google>` as an authentication method
+
+.. important::
+   - | **Installing or uninstalling apps, managing users, etc., is up to you.**
+     | As the administrator of your database, you are responsible for its usage, as you know best
+       how your organization works. This includes the installation or uninstallation of :ref:`apps
+       <general/apps-modules>` as well as the number of :doc:`users <general/users>` currently in
+       use.
+   - | **Odoo Apps have dependencies.**
+     | This means that you may have to install modules that you do not actively use to access some
+       features of Odoo you might need.
+   - | **Always test app installation/removal on a duplicate of your database.**
+     | This way, you can know what apps dependencies may be required or what data may be erased.
+
+.. seealso::
+   - :ref:`Online Database management: Duplicate a database <duplicate_online>`
+   - :ref:`On-premises Database management: Duplicate a database <duplicate_premise>`
+   - `Submit a Support Ticket <https://www.odoo.com/help>`_.
+
+.. _general/apps-modules:
+
+Applications and modules
+========================
+
+Odoo is *not a smartphone*, and its apps shouldn't be installed or uninstalled carelessly. Apply
+caution when adding or removing applications and modules on your database since this may impact your
+subscription costs.
+
+.. _general/uninstall-apps:
+
+Uninstall Apps
```

Comment:
```suggestion
Uninstall Apps
```
Sometimes you capitalize Apps, sometimes not. Not sure what's best but let's aim for consistency right? :)

Reply:
You're right, it should be "apps" here.
It is "Apps" when referring to the "Apps" button in the main menu.

Reply:
Also, I think that I should replace some "applications" with "apps", for consistency